documentation/book/ref-kafka-bridge-producer-configuration.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// assembly-kafka-bridge-configuration.adoc
+
+[id='ref-kafka-bridge-producer-configuration-{context}']
+= Kafka Bridge Producer configuration
+
+Kafka Bridge producer is configured using the properties in `KafkaBridge.spec.producer`.
+This property contains the Kafka Bridge producer configuration options as keys.
+The values can be one of the following JSON types:
+
+* String
+* Number
+* Boolean
+
+Users can specify and configure the options listed in the {ApacheKafkaProducerConfig} with the exception of those options which are managed directly by {ProductName}.
+Specifically, all configuration options with keys equal to or starting with one of the following strings are forbidden:
+
+* `ssl.`
+* `sasl.`
+* `security.`
+* `bootstrap.servers`
+* `group.id`
+
+IMPORTANT: The Cluster Operator does not validate keys or values in the `config` object provided.
+When an invalid configuration is provided, the Kafka Bridge cluster might not start or might become unstable.
+In this circumstance, fix the configuration in the `KafkaBridge.spec.producer.config` object, then the Cluster Operator can roll out the new configuration to all Kafka Bridge nodes.
+
+.Example Kafka Bridge producer configuration
+[source,yaml,subs="attributes+"]
+----
+apiVersion: {KafkaApiVersionPrev}
+kind: KafkaBridge
+metadata:
+  name: my-bridge
+spec:
+  # ...
+  producer:
+    config:
+      acks: 1
```

Comment:
Out of curiosity rather then to change this example ... does the `acks` field configure only for the whole bridge? Or can the producers override it? (cc @ppatierno)

Reply:
Historically, starting from bridging AMQP, the bridge has two different producers: one fixed at "0" as acks (whatever you set in the YAML, it will be overridden) which is called "settle mode" (from an AMQP perspective) and the other one which gets the acks you want from the configuration and it's called "unsettled mode" (which makes sense if you use "1" or "all" but .... not "0", which is anyway allowed to be configured but doesn't make any sense).